test/org/opensolaris/opengrok/web/DirectoryListingTest.java

```diff
@@ -84,11 +85,15 @@ private void create() throws Exception {
             assertTrue("Failed to set modification time",
                        file.setLastModified(val));
 
-            if (size > 0) {
-                FileOutputStream out = new FileOutputStream(file);
-                byte[] buffer = new byte[size];
-                out.write(buffer);
-                out.close();
+            try {
+                int s = Integer.parseInt(size);
+                if (s > 0) {
+                    FileOutputStream out = new FileOutputStream(file);
+                    byte[] buffer = new byte[s];
+                    out.write(buffer);
+                    out.close();
+                }
+            } catch (NumberFormatException e) {
```

Comment:
this is on purpose; if the size isn't numeric then nothing should be written to the file and it is a test so I suppose any logging isn't really necessary here.